cdk/lib/gudocs.ts

```diff
@@ -178,6 +178,7 @@ export class GuDocs extends GuStack {
 		});
 		scheduledLambda.addToRolePolicy(s3BucketPolicy)
 		scheduledLambda.addToRolePolicy(sharedParametersPolicy)
+		scheduledLambda.addToRolePolicy(pandaS3BucketPolcy)
```

Comment:
Can we add a comment to explain why this is necessary?

Reply:
Sure, I've added in the last commit.